src/app/mapView/MapView.jsx

```diff
@@ -32,41 +25,109 @@ export default class MapView extends Component {
     navigator.geolocation.getCurrentPosition(
       userPosition => {
         const { coords } = userPosition;
-        this.onCenterChanged({
-          lat: coords.latitude,
-          lng: coords.longitude,
+        this.setState({
```

Comment:
The original reason for onCenterChanged was that it both set state and re-fetched the data with the new center.

Is it that passing the new `center` to the map will trigger an onBoundsChanged anyway, and so it's not necessary to have an explicit fetch call?
I thought onBoundsChanged is only called when the change is triggered by interaction with the map, and not the props passed to it, but I don't really know.

Reply:
Yes, that is correct. onBoundsChanged will be called, so we don't need to explicitly re-fetch.